src/api/apiFetch.js

```diff
@@ -10,9 +10,11 @@ import { makeErrorFromApi } from './apiErrors';
 
 const apiVersion = 'api/v1';
 
-export const getFetchParams = <P>(auth: Auth, params: P): { headers: mixed, ...P } => {
```

Comment:
It looks like we've gestured toward not allowing `params` to contain
    `headers`, so, make that explicit.

I think the old code doesn't so much gesture toward `params` not having that property, as say that the return value will have that property. P could have that property, and then the return type would just have it with the same type as P.

But certainly the actual interface of this function is that you can't include `headers`, and it'd break the functionality if you did. So, glad to have that expressed in the type. :slightly_smiling_face: